src/routes/index.ts

```diff
@@ -148,6 +149,16 @@ router
   .get(trustInvolved.get)
   .post(trustInvolved.post);
 
+router
+  .route(config.TRUST_INVOLVED_URL + config.ID + config.TRUST_HISTORICAL_BENEFICIAL_OWNER_URL + config.ID + '?')
```

Comment:
Please add to config TRUST_ID parameter 'trustId' and BO_ID parameter 'boId'
build url 
```
config.TRUST_DETAILS_URL + config.TRUST_ID + config.TRUST_HISTORICAL_BENEFICIAL_OWNER_URL + config.BO_ID + '?'
```

Reply:
This is a good idea but I think it would end up being out of the scope of the ticket. Updating to config.BO_ID would require changes to other areas in the code referencing a BO's Id so that it is consistent throughout. 
I'll raise a tech debt ticket for this.